control/control.CASP.xml

```diff
@@ -122,6 +122,11 @@ textdomain="control"
 
         <!-- BNC #435479: Show support status in package details -->
         <display_support_status config:type="boolean">true</display_support_status>
+
+        <!-- AutoYaST clone will be called in the installed system. Unfortunately there is no information about handling recommended
+             packages while first installation. This setting will be taken for install_recommended in the software section
+             of AY configuration file. Default is "true" -->
```

Comment:
Well, it's not needed at all as we're using the number for the simplified workflow.